testing/mock/ibc_module.go

```diff
@@ -151,3 +154,7 @@ func (im IBCModule) OnTimeoutPacket(ctx sdk.Context, packet channeltypes.Packet,
 
 	return nil
 }
+
+func CreateOnRecvMockCapabilityName(packet channeltypes.Packet) string {
```

Comment:
Open for ideas for where to put this fn. I was considering creating a `keys.go`?

Reply:
Moving to `values.go` as similar fn already exists as pointed out by @colin-axner

Reply:
Actually, I've updated the tests that were using the helper in `values.go` to use the new helper in `mock/ibc_module.go` as we get a cyclical dependency issue trying to import `ibctesting` into `mock/ibc_module.go`. Otherwise, we duplicate the function.